### DISCUSSION TITLE
Opulous as an NFT market

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#282] [General]
Here comes an excerpt from the initial Discord discussion:

- - - - -
BisforBear in [Discord](https://discord.com/channels/906917846754418770/917476070092046366/925480706279366716) — 12/28/2021 :

> @ipaleka I haven't seen it addressed here yet, but should we consider Opulous? As I understand they intend to be an nft marketplace for musicians. Crescendo seems to be a funding site now, but could also become a marketplace...

- - - - -
ipaleka — 12/28/2021

Right now we're interested in the providers who have an onchain "shop" (NFT listed to buy/sell). Dunno if Opulous have any, if they have then they should be added.

- - - - -
BisforBear — 12/28/2021

I believe they have already minted and sold or given away a few songs already 

- - - - -
ipaleka — 12/28/2021

Ok, but bear in mind that I'd rather hard-code those few in our code than spending time in this phase on developing for something with such a small market share. 
We need those 3-4 NFT markets first, everything else is just for you to give people tasks. We need to start with the major one or the major 3-4.

- - - - -

Today Maru the Cat [in our Discord](https://discord.com/channels/906917846754418770/908054330265960478/1029810807749488711) emphasized the problem that the Opulous MFTs are under "Not evaluated" section.

- - - - -
It seems like the Opulous MFTs(music NFTs) are under “Not Evaluated.”  Maybe the MFTs could be categorized under Opulous MFTs?
![Opulous](https://user-images.githubusercontent.com/49662536/195417245-98e42d47-f1cf-4ce2-a2f7-cb54bea89180.png)

That’s how it shows on the Opulous site
Hmmm but it’s more like asset or token

The MFTs is a unique kind of category cause I think it’s more like a token vs a nft 
Like there is amount or unit value to the amount of MFTs you purchased at sale

- - - - -

ipaleka — Today at 7:50 PM

> The MFTs is a unique kind of category cause I think it’s more like a token vs a nft (edited)
> Like there is amount or unit value to the amount of MFTs you purchased at sale

So, Maru, there are no active sales right now to be checked for, how did you exactly buy those MFTs?

- - - - -

**Comment by marucat27** [comment]
These were sold through their partner sites wefunder and republic website.  Right now these cannot be traded or sold but will be available to do so in the future my understanding via their own marketplace/exchange.  Suppose better to implement into a category when all that is set up on their end?

**Reply by ipaleka** [reply]
Would you be able to research creator addresses for their NFTs/MFTs and check does each of them is created by a unique creator?

What's your approximation of how many of those Opulous MFTs have been minted so far?
